cmd/terratag/main.go

```diff
@@ -22,7 +23,11 @@ func main() {
 	}
 
 	if args.Version {
-		fmt.Printf("Terratag v%s\n", version)
+		var versionPrefix string
```

Comment:
with goreleaser it's without the v.
with ldflags it's with the v (I think...)...
Support both usecases: and a 'v' if not there.